packages/orchestration/src/types.ts

```diff
@@ -4,7 +4,6 @@ import type { ZCF } from '@agoric/zoe';
 
 export type * from './chain-info.js';
 export type * from './cosmos-api.js';
-export type * from './ethereum-api.js';
```

Comment:
consider leaving the file empty with a comment about potential Ethereum-only APIs